source/code/plugins/patch_management_lib.rb

```diff
@@ -411,10 +413,22 @@ def process_yum_record(record, tag, host, time)
 		@log.debug "LinuxUpdatesProgress: Parsing a record of type #{tag}"
 		update_run = []
 		update_run_record = {}
+
+        # 3 possible formats of package name:
+        # a: 'libXft-2.3.2-1.el6.x86$-1.28-12.el6.x86_64' -- starts with package name. and no dash in the middle
+        # b: '32:bind-utils-9.9.4-38.el7_3.1.x86_64' -- starts with 'digits:'
+        # c: 'sssd-client-1.14.0-43.el7_3.11.x86_64' -- starts with package name and has dash in the middle:
+        package_name_sections = record["package-name"].split(/-\d/)
+        package_name = nil;
+        if !package_name_sections.nil? && package_name_sections.length > 0
+            package_name = package_name_sections[0][/[\A\d:+]*(\S*)/, 1]
+        end
+        
 		update_run_record["Computer"] = host
 		update_run_record["OSType"] = "Linux"
 		update_run_record["UpdateRunName"] = getUpdateRunName()
-		update_run_record["UpdateTitle"] = record["package-name"]
+		update_run_record["UpdateTitle"] = ""
```

Comment:
There is no String with a value of nil. Because nil is simply not a String.  nil is actually also an object in Ruby. So I will keep initialize it with empty string.